wiki/osu!_wiki/Maintenance/en.md

```diff
@@ -0,0 +1,106 @@
+# osu! wiki maintenance
+
+*See also: [osu! wiki contribution guide](/wiki/osu!_wiki_Contribution_Guide)*
+
+This article describes technical or administrative aspects of the osu! wiki. It also covers maintenance procedures, which are required to keep it up to date—perhaps you may help with one of them. For all discussions of the wiki, use the `#osu-wiki` channel of the [osu!dev Discord server](/wiki/osu!dev_Discord_server).
+
+## Administrators
+
+*Main article: [List of the osu! wiki maintainers](/wiki/osu!_wiki/Maintenance/List_of_maintainers)*
+
+The administrators are people with the [collaborator level of access](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/permission-levels-for-a-user-account-repository#collaborator-access-for-a-repository-owned-by-a-user-account) to the [`ppy/osu-wiki`](https://github.com/ppy/osu-wiki/), where all the data is stored. They triage and label issues, manage open pull requests, and make decisions regarding the osu! wiki's present and future.
+
+Maintainers perform the final review of pull requests and merge them. If you have a contribution which needs attention, [mention one of them](/wiki/osu!_wiki/Maintenance/List_of_maintainers) in the `#osu-wiki` channel.
+
+## Technical details
+
+### Issue tracker
+
+The osu! wiki's [issue tracker](https://github.com/ppy/osu-wiki/issues) contains ideas and requests for possible improvements, both for the articles themselves and wiki-related parts of the website. Open an issue if you have a feature request for wiki, or if you found a mistake on one of the pages. Please note that **this is only limited to the osu! wiki**: if you need assistance with other official projects related to osu!, use their own issue trackers:
+
+- [osu!(lazer)](https://github.com/ppy/osu)
+- [The osu! website](https://github.com/ppy/osu-web/)
+- [Stable osu! build issues](https://github.com/ppy/osu-stable-issues)
```

Comment:
as-is slightly ambiguous and reads weirdly

```suggestion
- [osu!(stable) issues](https://github.com/ppy/osu-stable-issues)
```

Reply:
true. couldn't pinpoint what's wrong